README.md

```diff
@@ -42,17 +42,50 @@ In your CDN configuration, set the header defined in `BAY_INGRESS_HEADER` with t
 
 Once deployed, if the header is missing in the request nginx will return a `405 Not Allowed` HTTP response.
 
-# Contribute
+## Contribute
 [Open an issue](https://github.com/dpc-sdp/bay) on GitHub or submit a pull request with suggested changes.
 
-# Support
+## Image versioning and release management
+Bay image versioning follows an abstracted semantic versioning model.
+
+A major version is identified with a numeral and the minor version is abstracted to `x`. This provides flexibility for 
+future versioning requirements.
+
+In practice this looks like `1.x`, `2.x` etc.
+
+At any one time the two most recent tags are supported. These are identified based on numerical order.
+
+The lower value tag references a 'production safe' image.
+
+The higher value tag includes major updates to the image in testing.
```

Comment:
I think this is more to represent backwards breaking changes. In testing implies that you shouldn't move to the higher number until the next higher number is released. 

In the example; 1.x would only receive security updates and 2.x would receive non-breaking feature updates. Projects should aim to run the latest version available.

Reply:
>Projects should aim to run the latest version available.
👍 
When will we create the x.x-dev branches? And then the next major version?